tests/test_utils.py

```diff
@@ -255,3 +258,52 @@ def test_trim_commas_df(self, local_manifest):
         trimmed_df = df_utils.trim_commas_df(df_with_nans)
 
         assert_frame_equal(trimmed_df, local_manifest)
+
+
+class TestValidateUtils:
+
+    def test_validate_schema(self, schema_explorer):
+
+        se_obj = schema_explorer("tests/data/schema_org_schemas/example.jsonld")
```

Comment:
If you implement `get_schema_explorer()` from my other comment, you can add a fixture in this file called `schema_explorer` that takes `helpers` as an input and produces this specific schema explorer instance.